### DISCUSSION TITLE
`composer global ..` and `Do you want to re-run the command with --dev?`

### DISCUSSION THREAD

**Discussion by staabm** [composer/composer#12253] [General]
I think in composer global space there is not really a point whether deps are installed as `require-dev` or `require`.

wdyt about dropping this unecessary `Do you want to re-run the command with --dev?`- user interaction when running global commands?

**Comment by Seldaek** [comment]
https://github.com/composer/composer/pull/12268

**Reply by staabm** [reply]
Thanks!
